backend/posts/migrations/0001_initial.py

```diff
@@ -28,7 +28,7 @@ class Migration(migrations.Migration):
                 ('content', models.TextField(verbose_name='Post content')),
                 ('title', models.CharField(max_length=200, verbose_name='Post title')),
                 ('creation_datetime', models.DateTimeField(verbose_name='Post creation datetime')),
-                ('data_hash', models.BinaryField(max_length=32)),
+                ('data_hash', models.CharField(max_length=66)),
```

Comment:
Why are you editing migration file directly 😂 ?

Reply:
It's good habit to keep one migration per one PR and append only changes ;)

Reply:
Well, since nobody is using it, I just removed the old migrations and regenerated them xd

Reply:
I guess it's a bad practice when there is any kind of production or more than 1 person working on the project but it wasn't the case until recently.